common/config/azure-pipelines/templates/gather-docs.yaml

```diff
@@ -0,0 +1,61 @@
+# This template handles gathering all of the raw docs artifacts into a single staging directory
+# to allow them to be processed for publication.
+#
+# Note: This step has a dependency on the
+
+parameters:
+  - name: workingDir
+    type: string
+    default: $(System.DefaultWorkingDirectory)
+  - name: stagingDir
+    type: string
+    default: $(System.ArtifactsDirectory)/staging/
+
+steps:
+  ## Why is this needed??
+  - task: PublishPipelineArtifact@1

```

Comment:
@williamkbentley @BaoTon any idea why this was needed?  Is this to just debug it?

Reply:
I actually have no idea why that is needed - @williamkbentley any ideas?

Reply:
I have a feeling it might be related to the timestamp for pages that were untouched during the build.

Reply:
Okay, the timestamp I think should be the `.updated.json` that is lower down in the file.  This one just looks to be publishing all generated-docs data.